packages/template-retail-react-app/app/pages/cart/partials/cart-secondary-button-group.jsx

```diff
@@ -101,14 +101,16 @@ const CartSecondaryButtonGroup = ({
                             />
                         </Button>
                     )}
-                    {!variant.type?.item && (
-                        <Button variant="link" size="sm" onClick={() => onEditClick(variant)}>
-                            <FormattedMessage
-                                defaultMessage="Edit"
-                                id="cart_secondary_button_group.action.edit"
-                            />
-                        </Button>
-                    )}
+                    {/* Only show edit button if it's a standard product */}
```

Comment:
Please fix the comment. We're showing the edit button if it's _not_ standard product.